certified-connectors/Africa's Talking Voice/apiDefinition.swagger.json

```diff
@@ -0,0 +1,211 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Africa's Talking Voice",
+    "description": "Build dynamic scalable voice applications that reside entirely in the cloud.",
```

Comment:
@crispinkoech Please provide a detailed description which should describe concisely the main purpose and value offered by your connector

Refer - [Documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#submission-requirements)